doc/releases/changelog-dev.md

```diff
@@ -346,10 +320,10 @@ Users can specify the control wires as well as the values to control the operati
   Toffoli(wires=[2, 1, 0])
   ```
 
-* Adds caching to the `compute_matrix` and `compute_sparse_matrix` of simple non-parametric operations.
+* Added caching to the `compute_matrix` and `compute_sparse_matrix` of simple non-parametric operations.
   [(#3134)](https://github.com/PennyLaneAI/pennylane/pull/3134)
 
-* Add details to the output of `Exp.label()`.
+* Added details to the output of `Exp.label()`.
```

Comment:
Are the details worth mentioning here?

Reply:
Instead of `Exp`, the label is now `Exp({coeff} {base.label()})`. So for `qml.exp(2, qml.PauliX(0))` the label is `Exp(2, X)`